cpp/inc/bond/core/blob.h

```diff
@@ -416,7 +391,7 @@ inline T blob_cast(const blob& from)
 template <typename A>
 inline blob blob_prolong(blob src, const A& allocator)
 {
-    if (src._buffer)
+    if (src._buffer.use_count() != 0)
```

Comment:
Is using `use_count()` safe?

Reply:
Here we are checking for the ownership presence (i.e. not the actual reference count). That should be safe even for concurrent access.